tests/constraints/test_list_constraints.py

```diff
@@ -68,3 +70,10 @@ def test_handle_constrained_list_with_different_types() -> None:
         field = create_model_field(t_type, min_items=1)
         result = handle_constrained_collection(collection_type=list, model_field=field, model_factory=ModelFactory)
         assert len(result) > 0
+
+
+def test_handle_unique_items() -> None:
```

Comment:
The best stable test I was able to come up. Everything else was flaky requiring reruns to ensure uniqueness.

Bool has only two possible values so generating list of bools with `min_items=2` solves the problem